assets/prototype/domain/eventEditor/containers/queries/useFetchPriceTypes.js

```diff
@@ -1,44 +1,17 @@
 import { useQuery } from '@apollo/react-hooks';
-import { useEffect, useState } from '@wordpress/element';
 import { GET_PRICE_TYPES } from './priceTypes';
-import useToaster from '../../../../infrastructure/services/toaster/useToaster';
-import useStatus from '../../../../infrastructure/services/status/useStatus';
+import useInitToaster from '../../../../infrastructure/services/toaster/useInitToaster';
 
 const useFetchPriceTypes = () => {
 	console.log('%c useFetchPriceTypes: ', 'color: deeppink; font-size: 14px;');
-	const [initialized, setInitialized] = useState(false);
-	const { setIsLoading, setIsLoaded, setIsError } = useStatus();
 
-	const toaster = useToaster();
-	const toasterMessage = 'initializing price types';
-
-	const { data, error, loading } = useQuery(GET_PRICE_TYPES, {
-		onCompleted: () => {
-			toaster.dismiss(toasterMessage);
-			toaster.success(`price types initialized`);
-		},
-		onError: (error) => {
-			toaster.dismiss(toasterMessage);
-			toaster.error(error);
-		},
+	const { onCompleted, onError, initializationNotices } = useInitToaster({
+		loadingMessage: `initializing price types`,
+		successMessage: 'price types initialized',
 	});
 
-	useEffect(() => {
-		setIsLoading('priceTypes', loading);
-		setIsLoaded('priceTypes', !!data);
-		setIsError('priceTypes', !!error);
-	}, [data, error, loading]);
```

Comment:
This was not supposed to be removed.

Reply:
sry... I totally missed that in my review :(